js/src/edit.js

```diff
@@ -68,6 +68,28 @@ function sortComponentsByPosition( components ) {
 	} );
 }
 
+/**
+ * Render the metabox portal.
+ *
+ * @returns {null|ReactElement} The element.
+ */
+function renderMetaboxPortal() {
+	const metaboxElement = document.getElementById( "wpseo-meta-section-react" );
+
+	if ( metaboxElement ) {
```

Comment:
I would use the defensive way that returns null if metaboxElement does not exist.